### ISSUE TITLE
The Vuex State variables doesnt update. It just stays with the initial values. 

### ISSUE THREAD

**Issue by zuaaef** [vuejs/vuex#2046]
### Version
6.0.0-beta.11

### Browser and OS info
MacOS catalina 10.15.4 (19E266) with google chrome Version 90.0.4430.212 (Official Build) (x86_64)

### Steps to reproduce
I have to use console.log() as a workaround for me to see the values.  I just run the app, and update values in the Vuex Store.

### What is expected?
values in state should update

### What is actually happening?
values stay the same as default -> https://user-images.githubusercontent.com/18083923/119898167-77304980-bf0f-11eb-8d89-f076e2edec97.png

**Comment by 5z5** [comment]
how to push myself's code to the project

**Comment by Hendersonpinto** [comment]
Is this issue solved? I am using Nuxt and can't seem to get the state to update 😞

**Comment by jaitaiwan** [comment]
I have up and switched to Pinia

**Comment by RobSpectre** [comment]
Also encountering this problem. Using `vuex-persist`, not `vuex-persistedstate`.

**Comment by michug** [comment]
Same problem using using vuex-persist

"vue": "^3.2.27",
"vuex": "^4.0.2",
"vuex-persist": "^3.1.3"

**Comment by torressam333** [comment]
Still an issue to this day. The refresh icon in vue dev tools does nothing. If I manually update a Boolean for example from true to false in an attempt to toggle ui functionality, it does nothing. Eeeesh been a problem forever I see.

**Comment by patchthecode** [comment]
i went to pinia

**Comment by Cheglok** [comment]
same problem
"vue": "^3.2.26",
    "vuex": "^4.0.0-0",
    "vuex-persist": "^3.1.3"

**Comment by zuaaef** [comment]
this has started workinng again for me. Not sure how or when. Was it a restart? donno.

**Comment by MarcelFinkbeiner** [comment]
Yes for me it is working too, unless I do anything with replaceState

**Comment by dovepa** [comment]
Same bug
